TreeTracker/UI/Views/Authentication/SignIn/SignInViewController.swift

```diff
@@ -23,24 +24,37 @@ class SignInViewController: UIViewController, KeyboardDismissing, AlertPresentin
             usernameTextField.returnKeyType = .done
             usernameTextField.placeholder = L10n.SignIn.TextInput.PhoneNumber.placeholder
             usernameTextField.validationState = .normal
+            usernameTextField.backgroundColor = .black
+            usernameTextField.layer.borderColor = UIColor.lightGray.cgColor
+            usernameTextField.layer.cornerRadius = 10.0
+            usernameTextField.layer.borderWidth = 1.0
+            usernameTextField.clipsToBounds = true
         }
     }
     @IBOutlet private var usernameSegmentedControl: UISegmentedControl! {
```

Comment:
We should also set the text on the segmented control here too. It should be added tho the Localizable.strings file so swiftgen can generate the L10n enums which we can the use here.

Reply:
What part of the segmented controller should I set the text ?

![Simulator Screen Shot - iPhone 13 - 2021-11-01 at 11 29 00](https://user-images.githubusercontent.com/55071531/140403141-65f9ec0f-f1f1-4a58-a840-c38968a0a5f1.png)
?

Reply:
Forogt these had to text - ignore this